.gitignore

```diff
@@ -2,3 +2,5 @@ pkg/*
 **/.DS_Store
 .rspec
 gemfiles
+.ruby-version
```

Comment:
I think we'd like to keep this in as it's helpful to ensure development against the latest working version of ruby.  Objections?  I'm curious what issues you had with this in place.